gestures.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: Gestures
+permalink: /gestures/
+---
+
+The gesture system in Flutter has two separate layers.  The first layer has raw
+pointer events, which describe the location and movement of pointers (e.g.,
+touches, mice, and styli) across the screen. The second layer has gestures,
+which describe semantic actions that consist of one or more pointer movements.
+
+# Pointers
+
+Pointers represent raw data about the user's interaction with the device's
+screen.  There are four types of pointer events:
+
+- `PointerDownEvent` The pointer has contacted the screen at a particular
+   location.
+- `PointerMoveEvent` The pointer has moved from one location on the screen to
+   another.
+- `PointerUpEvent` The pointer has stopped contacting the screen.
+- `PointerCancelEvent` The framework is no longer delivering events for this
```

Comment:
The cancel one is not quite accurate. It's more that the system has decided to stop telling the framework about the pointer. Nothing in the framework itself generates the event, even in the case of a widget being taken out of the flow in some way and thus no longer getting events for the pointer.